conf/hosts.conf

```diff
@@ -0,0 +1,7 @@
+#
+# hosts file for the CUBRID, 2022-06-17 (webmaster@cubrid.com)
+#
+127.0.0.1	localhost
+192.168.0.1	node_1       # Seoul
+192.168.32.2	node_2       # Daejeon
+10.168.0.254	node_1B      # Backup Center
```

Comment:
If is it this real IP, please removed this.
It needs an example input is required in this file.

Reply:
Ok, I understand it. Thank you.